xbmc/windowing/X11/WinSystemX11GLContext.cpp

```diff
@@ -287,7 +287,7 @@ bool CWinSystemX11GLContext::RefreshGLContext(bool force)
         m_vaapiProxy.reset(X11::VaapiProxyCreate());
         X11::VaapiProxyConfig(m_vaapiProxy.get(), GetDisplay(),
                               static_cast<CGLContextEGL*>(m_pGLContext)->m_eglDisplay);
-        bool general, deepColor;
+        bool general=false, deepColor;
```

Comment:
Why do you initialize only one of them?

Reply:
the other is only used if general is true, which means it was set. so it isn't necessary. i can do it if you prefer it for OCD reasons or whatever ;)

Reply:
You have to look into the function to know that. Also in the future it may change.

Reply:
done.

Reply:
To be honest this doesn't make much sense to me here :/ `VAAPIRegisterRender` always initializes `general` to `false`, as it can be expected to. This confusion is one of the reasons I detest reference out parameters in general though.

Reply:
it does not if you have no vaapi renderer available.

Reply:
Acknowledged. Putting the variable initialization there would make just as much sense I guess, so fine by me.